src/main/java/com/diffmin/App.java

```diff
@@ -209,6 +211,13 @@ private static void applyInsertion(ImmutableTriple<Integer, CtElement, CtElement
                 ((CtExecutable<?>) inWhichElement)
                         .addParameterAt(where, (CtParameter<?>) toBeInserted);
                 break;
+            case THROWN:
+                ((CtExecutable<Throwable>) inWhichElement)
+                        .addThrownType((CtTypeReference<Throwable>) toBeInserted);
```

Comment:
I expected this to add the `ThrownType`s and automatically order them based on the comparator written but apparently, it just appends the type references which causes the test to fail.

Also, I had to [suppress a warning](https://github.com/SpoonLabs/diffmin/pull/56/files#diff-35cb1714dcacdffe9411aacaae85c435bdf37c26792ebcf7f3b4a24e6f84942cR188) because of this invocation.

Reply:
As discussed during the meeting, this is due to the position taking precedence in sorting.